docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -e
+
+# wait shortly and then run db migrations (retry on error)
+while ! ./manage.py migrate 2>&1; do
+  echo "⏳ Waiting on DB..."
+  sleep 3
+done
+
+# create superuser silently
+if [ -z ${SUPERUSER_NAME+x} ]; then
+  SUPERUSER_NAME='admin'
+fi
+if [ -z ${SUPERUSER_EMAIL+x} ]; then
+  SUPERUSER_EMAIL='admin@example.com'
+fi
+if [ -z ${SUPERUSER_PASSWORD+x} ]; then
+  if [ -f "/run/secrets/superuser_password" ]; then
+    SUPERUSER_PASSWORD="$(< /run/secrets/superuser_password)"
+  else
+    SUPERUSER_PASSWORD='admin'
+  fi
+fi
+if [ -z ${SUPERUSER_API_TOKEN+x} ]; then
+  if [ -f "/run/secrets/superuser_api_token" ]; then
+    SUPERUSER_API_TOKEN="$(< /run/secrets/superuser_api_token)"
+  else
+    SUPERUSER_API_TOKEN='0123456789abcdef0123456789abcdef01234567'
```

Comment:
I just realized we need to make sure we set the `SUPERUSER_API_TOKEN` in the helm chart, as this is not a very safe thing